internal/pipe/ko/ko.go

```diff
@@ -0,0 +1,269 @@
+package ko
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"golang.org/x/tools/go/packages"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+	"github.com/google/ko/pkg/build"
+	"github.com/google/ko/pkg/commands/options"
+	"github.com/google/ko/pkg/publish"
+	"github.com/goreleaser/goreleaser/internal/ids"
+	"github.com/goreleaser/goreleaser/internal/semerrgroup"
+	"github.com/goreleaser/goreleaser/internal/tmpl"
+	"github.com/goreleaser/goreleaser/pkg/config"
+	gcontext "github.com/goreleaser/goreleaser/pkg/context"
```

Comment:
usually what we do is use this one as `context` and the regular one as `stdctx` or `stdcontext`, as in all other places `context` is the goreleaser context...